CMakelists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.2)
+project(cppfront)
+
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY 					${CMAKE_BINARY_DIR}/bin)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY 					${CMAKE_BINARY_DIR}/bin)
+
+# Until CMake 3.20.3, if you ask for C++20, using set(CMAKE_CXX_STANDARD 20), you'll get -std:c++latest.
+# Since CMake 3.20.4, set(CMAKE_CXX_STANDARD 20) gets you -std:c++20,
+# so you need set(CMAKE_CXX_STANDARD 23) to get -std:c++latest --
+# assuming that your MSVC compiler version is 16.11 Preview 1 or later (see cmake commit 3aaf1d91bf353).
+# issue with the standard library of MSVC
+# https://stackoverflow.com/questions/64889383/how-to-enable-stdclatest-in-cmake
+set(CMAKE_CXX_STANDARD 23)
+
+set(SRC_FILES
+	"source/cppfront.cpp"
+)
+
+add_executable(${PROJECT_NAME} ${SRC_FILES})
+target_include_directories(${PROJECT_NAME} PUBLIC "include")
```

Comment:
Compiling `cppfront` doesn't require this.

Reply:
Can you please clarify what is not required?

Reply:
Line 20. You can compile `cppfront` without `cpp2util.h`.